typing/includemod.ml

```diff
@@ -1059,25 +1060,12 @@ module Functor_inclusion_diff = struct
         let arg2 = Subst.modtype Keep st.subst arg2 in
         let env = Env.add_module id2 Mp_present arg2 env in
         expand_params { st with env }
-    | Keep (Named (name1, _), Named (name2, arg2), _) -> begin
-        let arg' = Subst.modtype Keep st.subst arg2 in
-        match name1, name2 with
-        | Some id1, Some id2 ->
-            (* two matching abstract parameters: we add one identifier to the
-               environment and record the equality between the two identifiers
-               in the substitution *)
```

Comment:
Could we preserve this comment? I found it helpful.

Reply:
Good point, I will add the comment to the new shared function.